packages/ia-components/sandbox/details/CrawlConfig.js

```diff
@@ -113,6 +114,7 @@ export default class CrawlConfig extends IAReactComponent {
         }
       });
     }
+    ev.preventDefault(); // Dont propogate event
```

Comment:
adding another comment around prevent default - do we need to do this?

Reply:
This one could go either way, I thought it was preferential not to pass it up when handled, but happy to remove (in this case, not in the Anchor cases).